lib/modules/dosomething/dosomething_user/dosomething_user.install

```diff
@@ -143,3 +143,11 @@ function dosomething_user_update_7012() {
   field_delete_field('field_school_id');
   field_delete_field('field_user_registration_source');
 }
+
+/**
+ * Remove IPs from sessions table.
+ */
+function dosomething_user_update_7013()
+{
+  db_drop_field('sessions', 'hostname');
```

Comment:
Actually, one more question - does this remove the column altogether or just zero it out? Are we safe to remove, or will Drupal continue to try to write to this?